ext/telemetry/errors.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+module.exports = function(tracker) {
+  return {
+    trackUncaught: () => {
+      window.onerror = (msg, url, line, col, err) => {
+        var data = {
```

Comment:
Atom is supporting node 6 so we can use `const`, `let`, and the shorthand Object literal from ES6: 

```js
const data = {msg, url, line, col}
```